### PR TITLE
[bi] add button link bi

### PR DESCRIPTION
**Pull request by danarish4**

### CODE REVIEW

File: packages/plugin-button/web/src/toolbar/buttonInputModal.jsx [hunk 1]
```diff
@@ -65,13 +72,29 @@ export default class ButtonInputModal extends Component {
...
+      settings: { rel, target, url },
+      validUrl,
+    } = this.state;
+    validUrl &&
+      this.props.helpers?.onPluginAction?.(ADD_PLUGIN_LINK_BI, {
+        plugin_id: LINK_BUTTON_TYPE,
+        link: url,
+        newTab: isNewTab(target),
+        category: WEB_ADDRESS_CATEGORY,
+        nofollow: rel,
```
Comment (maimonav) [inline packages/plugin-button/web/src/toolbar/buttonInputModal.jsx:86]: `rel` value contains 'nofollow', so it can lead to missinformation.

**Review by maimonav — APPROVED** [review]
Looks good, just take a look on the `rel` comment. :)

File: packages/plugin-button/web/src/toolbar/buttonInputModal.jsx (hunk 1 above)
Comment (maimonav) [reply packages/plugin-button/web/src/toolbar/buttonInputModal.jsx:86]: IMO, the BI event need to be altered, having `rel` param instead of `nofollow`.

Comment (danarish4) [reply packages/plugin-button/web/src/toolbar/buttonInputModal.jsx:86]: I checked all 'rel' values and its always set to true/false

**Pull request merged**